src/plugins/Camera.js

```diff
@@ -58,8 +58,7 @@ module.exports = class Camera extends OverlayPlugin {
             this.toolbarBtn.className += ' gm-disabled-widget-pop-up';
             this.toolbarBtnImage.className += ' gm-disabled-widget-icon';
         } else {
-            // TODO remove ? from this.instance.mediaManager?.toggleVideoStreaming and refacto to avoid error
-            this.toolbarBtn.onclick = this.instance.mediaManager?.toggleVideoStreaming;
+            this.toolbarBtn.onclick = () => this.instance.mediaManager.toggleVideoStreaming();
```

Comment:
it could be nice for this one to use `this.instance.addListener` so that it is autoremoved at player destruction